### ISSUE TITLE
Fix accessibility issues for right sidebar navigation

### ISSUE THREAD

**Issue by ekrimmel** [paleo-data/knowledge-hub#90]
Basically the right sidebar is nested inside the main div but should be at the same level. Here is what the accessibility checker provides for the error:
 
> A complementary landmark, typically represented by the element or role="complementary", is meant to hold ancillary content that complements the main content on the page. To maintain a clear structure for screen readers and other assistive technologies, complementary landmarks must not be nested inside other ARIA landmarks like role="main" or role="contentinfo. This ensures easier navigation for users relying on assistive technologies.

 _Originally posted by @adamancer in [#54](https://github.com/paleo-data/knowledge-hub/issues/54#issuecomment-3176255519)_
